### PR TITLE
Redis deployment

### PR DESCRIPTION
**Pull request by bj2k61**
How to review 
cd into teaching-vacancies  make review_aks terraform-init-aks pr_id=postgres VERBOSE=1
make review_aks terraform-plan-aks pr_id=postgres VERBOSE=1

### CODE REVIEW

File: terraform/aks/database.tf [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  azure_resource_prefix        = var.azure_resource_prefix
+  service_short                = var.service_short
+  config_short                 = var.config_short
+  service_name                 = var.service_name
+  cluster_configuration_map    = module.cluster_data.configuration_map
+  use_azure                    = var.deploy_azure_backing_services
+  azure_enable_monitoring      = var.enable_monitoring
+  azure_patch_schedule         = [{ "day_of_week": "Sunday", "start_hour_utc": 01 }]
+  azure_maxmemory_policy       = "allkeys-lfu"
+}
```
Comment (RMcVelia) [inline terraform/aks/database.tf:29]: there are 2 redis instances for tv. cache and queue
e.g.
teaching-vacancies-redis-cache-review-pr-6175
teaching-vacancies-redis-queue-review-pr-6175
the queue instance requires `maxmemory policy     : noeviction`

File: vendor/.keep
Comment (RMcVelia) [inline vendor/.keep:1]: not sure why this is being deleted

File: terraform/aks/database.tf (hunk 1 above)
Comment (bj2k61) [reply terraform/aks/database.tf:29]: the parameter has been changed as recommended

File: vendor/.keep
Comment (bj2k61) [reply vendor/.keep:1]: resolved

File: global_config/review.sh [hunk 2]
```diff
@@ -1,4 +1,4 @@
-CONFIG=review_aks
+CONFIG=review
 var_file=${CONFIG}
```
Comment (nickdevdfe) [inline global_config/review.sh:2]: this needs to be hardset

Comment (bj2k61) [reply global_config/review.sh]: this has been updated

**Comment by bj2k61** [comment]
[Deploy redis](https://trello.com/c/88kVZPE3/529-deploy-redis)

**Pull request merged**